hibike/README.md

```diff
@@ -152,7 +152,9 @@ Message ID Enumeration:
 |  0x12   |   Subscription Response  |
 |  0x13   |       Device Read        |
 |  0x14   |       Device Write       |
-|  0x15   |      Device Data         |
+|  0x15   |       Device Data        |
+|  0x16   |    Heart Beat Request    |
```

Comment:
0x16 as Heart Beat Request won't work, since 0x16 has been allocated to the "Disable" in the newly-merged disable commits.

Please reallocate to 0x16 and 0x17